### DISCUSSION TITLE
本人自己另外拉了个分支增加了书源

### DISCUSSION THREAD

**Discussion by BaphometWei** [freeok/so-novel#64] [Show and tell]
已有书源因为防盗缘故大部分是乱码，所以自己另外加了书源，并且由于该书源不兼容原有代码，所以自己拉了个分支修改了下
该分支项目地址：https://github.com/BaphometWei/novel_download.git

**Comment by freeok** [comment]
[v1.7.3](https://github.com/freeok/so-novel/releases/tag/v1.7.3) 已支持69書吧

**Comment by freeok** [comment]
该书源的缺点是对 IP 进行了限流，即便能下载，速度也会慢非常之多，稳妥下载一章大约需要 2 ~ 3 秒
